jaxmarl/environments/jaxnav/jaxnav_singletons.py

```diff
@@ -708,11 +708,6 @@ def make_jaxnav_singleton_collection(collection_id: str, **env_kwargs) -> Tuple[
         "NarrowChicane2b",
         "Chicane4",
     ],
-    "new": [
```

Comment:
Why did you remove these tests? Ideally tests should be fixed unless you are removing a feature

Reply:
oh these were just used during development so tidying them into a clearer set